2015/proposals/banaszkiewicz-piotr-amy.md

```diff
@@ -0,0 +1,214 @@
+# Enhance Amy, a workshop-management platform for Software Carpentry
+
+## Abstract
+
+The number of Software Carpentry's workshops run weekly dynamically grows. All
```

Comment:
Maybe “is growing rapidly” instead of  “dynamically grows”?  I know Greg publishes charts of students-taught in the blog.  I don't know if he publishes charts of workshops-held, but linking to either of those from here would be nice.